R/modified_update.R

```diff
@@ -37,22 +54,54 @@ modified_update_calculate <- function(con, touchstone_name_mod, touchstone_use)
     "[%s] %s",
     ifelse(meta$group$activity_type == "routine", "Rout", "SIA"),
     meta$group$vaccine)
-
+  
   ## This is quite slow to run but we can just pull it once:
   fmt <- "SELECT * FROM v_target_pop_%s WHERE YEAR BETWEEN %d AND %d"
   pop <- list(
     total = DBI::dbGetQuery(con, sprintf(fmt, "total", year_min, year_max)),
     routine = DBI::dbGetQuery(con, sprintf(fmt, "routine", year_min, year_max)))
-
-
+  
+  
   index <- meta$group$index
   data <- lapply(index, function(i) mu_build_data(con, i, meta, pop))
-
+  
   data <- rbind_simple(data)
   ## Perform the update itself the update:
-  data$deaths_averted_new <- mu_scale("deaths_averted", data)
-  data$cases_averted_new <- mu_scale("cases_averted", data)
+  data$deaths_averted_new <- mu_scale("deaths_averted", data, method)
+  data$cases_averted_new <- mu_scale("cases_averted", data, method)
+  
   meta$data <- data
+  
+  if(version == "v2") {
```

Comment:
sorry to be pedantic, but can you keep a space between `if` and `(` here and elsewhere (similarly for `for`, etc)

Reply:
Done here and else where.